### DISCUSSION TITLE
Luci tramite MQTT non cambia stato in home assistant

### DISCUSSION THREAD

**Discussion by angeloprata** [indomus/forum#2389] [HUB: Home Assistant]
buongiorno, ieri ho installato Home Assistant da 0, ho 54 Shelly con Tasmota per le luci, tapparelle e termostati.
funziona tutto solo che le luci quando cambiano lo stato non comunicano tale cambiamento!

**Comment by snakuzzo** [comment]
Ok, ma ti avevo chiesto in chat di pubblicare qualche dettaglio in più.
Come hai configurato il device Tasmota ? Sei sicuro di averli aggiornati tutti all'ultima release ?
Se usi integrazione MQTT usi `SetOption19 1`, ma resta un'integrazione "sconsigliata" dagli stessi dev tasmota, i quali consigliano di utilizzare l'integrazione "Tasmota" (che si basa sempre su MQTT). In tal caso devi impostare `SetOption19 0`. 

Qui un paio di link ufficiali...

https://www.home-assistant.io/integrations/tasmota/
https://tasmota.github.io/docs/Home-Assistant/

**Reply by angeloprata** [reply]
ho attivato
setoption 19 e 30
switchmode3 1
tutto qua...cosi come li avevo prima di formattare...e funzionava tutto perfettamente
